packages/rn-tester/android/app/src/main/java/com/facebook/react/uiapp/RNTesterApplication.kt

```diff
@@ -135,7 +134,7 @@ class RNTesterApplication : Application(), ReactApplication {
     @UnstableReactNativeAPI
     override val reactHostInterface: ReactHost
         get() {
-            if (reactHost == null) {
+            if (!::reactHost.isInitialized) {
```

Comment:
This also can be refactored to use `by lazy{}` and you don't need the `lateinit` property anymore.

Also please rebase as this field has been renamed from `reactHostInterface` to `reactHost` just yesterday

Reply:
Yeah that makes much more sense to me, done